pantheon/src/main/java/tech/pegasys/pantheon/cli/PantheonCommand.java

```diff
@@ -487,6 +500,13 @@ private WebSocketConfiguration webSocketConfiguration() {
     return webSocketConfiguration;
   }
 
+  private PermissioningConfiguration permissioningConfiguration() {
+    final PermissioningConfiguration permissioningConfiguration =
```

Comment:
We need to validate that all bootnodes have been included in the nodes whitelist. I wonder if this would be the place to validate this...